patroni/ctl.py

```diff
@@ -70,6 +70,10 @@ def load_config(path, dcs):
 
     if not (os.path.exists(path) and os.access(path, os.R_OK)):
         logging.debug('Ignoring configuration file "%s". It does not exists or is not readable.', path)
+        if path == CONFIG_FILE_PATH:
+            ctx = click.get_current_context()
+            logging.warning('Default configuration file "%s" does not exist or is not readable, '\
+                    'try running "%s configure"', path, ctx.command_path)
```

Comment:
Meh, I would actually recommend to use the same config file as for patroni...

Reply:
Do you mean in the message or by default?

Reply:
It is a long story.
Originally patronictl had its own config file and didn't support the config file used by patroni.
Later it was changed, but the support of the dedicated file wasn't removed.
Maybe it is better to either "deprecate" the 'configure' option or make it create the symlink from the patroni.yaml to `~/.config/patroni/patronictl.yaml`

Reply:
I believe 'configure' is useful, it the easiest way to make a default config, I rewrote configure to create a symlink to patroni config

Reply:
Actually... Maybe we should rethink the workflow.
There is an very old issue opened by your collegue, https://github.com/zalando/patroni/issues/391
Have you seen it?

Reply:
I'm not sure I understand, if we already have a running cluster and we want to manage it with patronictl, what would be the workflow?

Reply:
If I wanted to run patronictl to monitor several clusters, without having a copy of a patroni config file of any member of each of the clusters, then I'd create a patronictl config file like this:

/very/short/config,yml
```yaml
namespace: /service/nonstandard/
etcd:
  hosts: 10.77.0.111:2379,10.77.0.112:2379,10.77.0.113:2379
  # below user in etcd has a role that allows him only to read from everything in /service/nonstandard
  username: patronictl_monitoring
  password: snakeoil
```
The file could also just contain the details on how to connect to any other DCS.
That (and possibly namespace and scope) should be the only configuration items of the patroni YAML file that are relevant to patronictl anyway.

Then I'd call `patronictl -c /very/short/config,yml list $clustername` for each clustername that I know.


A nice feature, further down the line, would be to have patronictl detect clusters in the given namespace on its own and print the member list for all of them...

Reply:
So I'd change the suggestion message to something like:
>please provide the patronictl configuration file using the --config-file or -c flags . You can simply use the patroni config file that you used to start one of the patroni cluster's members, or write a custom patronictl config file to be able to monitor and manipulate several patroni clusters with the same patronictl config file.